packages/machines/checkbox/src/checkbox.connect.ts

```diff
@@ -29,7 +29,7 @@ export function connect<T extends PropTypes>(state: State, send: Send, normalize
 
   const trulyDisabled = isDisabled && !isFocusable
 
-  const stateView = state.hasTag("checked") ? "checked" : "unchecked"
+  const stateView = state.value || "unchecked"
```

Comment:
This is leaking the state directly into the view. In the event we add more state nodes, it'll show up in the `state.value` here. Let's be a bit more explicity

```suggestion
  const stateView = state.matches("checked") ? "checked" : "unchecked"
```